docs/core/compatibility/sdk/8.0/unlisted-versions.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Breaking change: Unlisted packages not installed by default"
+description: Learn about a breaking change in the .NET 8 SDK where `dotnet tool install` no longer installs tools from unlisted versions of NuGet packages.
+ms.date: 11/07/2023
+---
+# Unlisted packages not installed by default
```

Comment:
is it worth adding `'for .NET Tools'` to this title? I'm ambivalent, but the specificity might help not alarm readers as much?